packages/core/src/components/TextField/Styled/InnerWrapper/InnerWrapper.styled.tsx

```diff
@@ -101,18 +101,27 @@ const getHeight = ({ size, theme, minRows, multiline, variant }: InnerWrapperPro
     const smallPadding = 1.1 * 2;
     const medPadding = 1.6 * 2;
 
+    const rows = minRows || 1;
```

Comment:
you can add 1 as default prop of the component.

Reply:
Updated: 086277f